pkg/controllers/termination/eviction.go

```diff
@@ -103,6 +103,7 @@ func (e *EvictionQueue) evict(ctx context.Context, nn types.NamespacedName) bool
 		return true
 	}
 	if err != nil {
+		logging.FromContext(ctx).Debugf("Could not evict pod %s due to an unexpected error: %s", nn.String(), err)
```

Comment:
This won't print the error string cleanly. We can also simplify this message.
```suggestion
		logging.FromContext(ctx).Debugf("Could not evict pod %s, %s", nn.String(), err.Error())
```

While we're in here, can we move the `.` from the other log lines?

Reply:
Sure thing!

Reply:
both `nn.String(), err.Error()` can be replaced with `nn, err` since they have .String they will both turn into strings.

Reply:
Updated commit with this change.